cicd/pipelines/automatic-pruning-taskrun-pipelinerun.adoc

```diff
@@ -0,0 +1,22 @@
+[id="automatic-pruning-taskrun-pipelinerun"]
```

Comment:
Would you apply the new Jupiter apply to this document?
https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist